### ISSUE TITLE
Protection Service dies silently in case of missing email

### ISSUE THREAD

**Issue by roscisz** [roscisz/TensorHive#173]
When user e-mail is missing, mailer fails (which is good), but the protection service no longer scans for violations:

```
  File "/home/tensorhive/TensorHive/tensorhive/core/services/ProtectionService.py", line 199, in do_run
    handler.trigger_action(violation_data)
  File "/home/tensorhive/TensorHive/tensorhive/core/violation_handlers/ProtectionHandler.py", line 8, in trigger_action
    self._protection_behaviour.trigger_action(*args, **kwargs)
  File "/home/tensorhive/TensorHive/tensorhive/core/violation_handlers/EmailSendingBehaviour.py", line 82, in trigger_action
    self._email_intruder(intruder_email, violation_data, timer)
  File "/home/tensorhive/TensorHive/tensorhive/core/violation_handlers/EmailSendingBehaviour.py", line 129, in _email_intruder
    self.mailer.send(email)
  File "/home/tensorhive/TensorHive/tensorhive/core/utils/mailer.py", line 76, in send
    self.server.sendmail(message.author, message.recipients, message.body)
  File "/usr/lib/python3.5/smtplib.py", line 876, in sendmail
    raise SMTPRecipientsRefused(senderrs)
smtplib.SMTPRecipientsRefused: {'<email_missing>': (553, b'5.1.2 The recipient address <email_missing> is not a valid RFC-5321 address. g5sm4044005ljk.59 - gsmtp')}
```

**Comment by roscisz** [comment]
It is still a thing. If smtp.sendmail() raises an exception, mailbot dies.

**Comment by roscisz** [comment]
fixed in 3310e5ab23bcc9537b741e3ae0135c6abc333395
